sdk/keyvault/azure-keyvault-keys/tests/test_crypto_client.py

```diff
@@ -104,6 +106,15 @@ def _to_bytes(hex):
         self._validate_rsa_key_bundle(imported_key, client.vault_url, name, key.kty, key.key_ops)
         return imported_key
 
+    def _import_symmetric_test_key(self, client, name):
+        key = JsonWebKey(
+            kty="oct-HSM",
+            key_ops=["encrypt", "decrypt", "wrapKey", "unwrapKey"],
+            k=bytes.fromhex("e27ed0c84512bbd55b6af434d237c11feba311870f80f2c2e3364260f31c82c8"),
```

Comment:
```suggestion
            k=binascii.unhexlify("e27ed0c84512bbd55b6af434d237c11feba311870f80f2c2e3364260f31c82c8"),
```

Reply:
I was just about to push a switch to codecs.decode -- is binascii preferred?

Reply:
No. I like it better than `codecs.decode(..., "magic string")` but certainly not enough to wrassle over it 💪

Reply:
I agree that the codecs approach isn't my favorite -- I'll make a note of this for when I give the tests a refresh 🤠